pages/documents/DeveloperTools/FaaS/Monitoring/logs.md

```diff
@@ -85,6 +83,15 @@ function lambda(input, callback) {
 	callback(null, 'Function has finished...');
 }
 ```
+<div class="important">
+<ul>
+<li>This function allows you to log sensitive information since there's no sanitation or limitations on the string you pass to the method! Please be considerate with what is logged and don't pass any sensitive information to this function, e.g a token or password.</li>
+<li>Logs are only persistent without a <b>Debug</b> level on external invocation.</li>
+<li>Logs with a <b>Debug</b> level will only be shown on manual invocation</li>
+<li>Logs in external invocations are limited to 10 entries, with an overall maximum of 5000 characters per lambda invocation. If this limit is exceeded, only one error log with the excess information will be shown, alongside the reason for exceeding the limit.</li>
+<li>Logs are only kept for 30 days. Older logs will be deleted.</li>
+</ul>
+</div>
 
 ### Accessing persistent Logs
```

Comment:
Please rephrase "The picture below shows the **Logs** section" => Rather explain directly the logs section.